zone/command.cpp

```diff
@@ -14106,6 +14107,135 @@ void command_network(Client *c, const Seperator *sep)
 	}
 }
 
+void command_viewzoneloot(Client *c, const Seperator *sep)
+{
+	std::map<uint32,ItemList> zone_loot_list;
+	std::list<NPC*> npc_list;
+	uint32 loot_amount = 0, loot_id = 1, search_item_id = 0;
+	if (sep->argnum == 1 && sep->IsNumber(1)) {
+		search_item_id = atoi(sep->arg[1]);
+	} else {
+		c->Message(
+			Chat::Yellow,
+			"Usage: #viewzoneloot [item id]"
+		);
+		return;
+	}
+	entity_list.GetNPCList(npc_list);
```

Comment:
https://github.com/EQEmu/Server/blob/f0d0c837108e6925103b268e33d4792ecdc9fe23/zone/entity.h#L514

Use that instead.